packages/cloudflare/src/api/overrides/incremental-cache/regional-cache.ts

```diff
@@ -27,6 +35,12 @@ type Options = {
   shouldLazilyUpdateOnCacheHit?: boolean;
 };
 
+interface PutToCacheInput {
+  requestKey: Request;
```

Comment:
Maybe pass `entryKey` and `isFetch` instead of `requestKey`?

`requestKey` could be changed to a string, would be nice to apply `.replaceAll(/\/+/, "/")` as we do in other cache handler because keys start with '/'

Reply:
Changed